src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -861,7 +861,7 @@
         az webapp config set -g MyResourceGroup -n MyUniqueApp --always-on true
   - name: turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
     text: >
-        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations "{"alwaysOn": true}"
+        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations '{"alwaysOn": true}'
```

Comment:
this only works for bash, maybe we can do `"{\"alwaysOn\": true}"`, also you need to update line 862 as well

Reply:
yes will do

Reply:
> 
> 
> yes will do

Nice to meet you